parts/k8s/kubernetesmastervars.t

```diff
@@ -88,20 +88,24 @@
     "kubernetesCcmImageSpec": "[parameters('kubernetesCcmImageSpec')]",
     "kubernetesAddonManagerSpec": "[parameters('kubernetesAddonManagerSpec')]",
     "kubernetesAddonResizerSpec": "[parameters('kubernetesAddonResizerSpec')]",
+{{if .OrchestratorProfile.KubernetesConfig.IsDashboardEnabled}}
     "kubernetesDashboardSpec": "[parameters('kubernetesDashboardSpec')]",
     "kubernetesDashboardCPURequests": "[parameters('kubernetesDashboardCPURequests')]",
     "kubernetesDashboardMemoryRequests": "[parameters('kubernetesDashboardMemoryRequests')]",
     "kubernetesDashboardCPULimit": "[parameters('kubernetesDashboardCPULimit')]",
     "kubernetesDashboardMemoryLimit": "[parameters('kubernetesDashboardMemoryLimit')]",
+{{end}}
     "kubernetesExecHealthzSpec": "[parameters('kubernetesExecHealthzSpec')]",
     "kubernetesDNSSidecarSpec": "[parameters('kubernetesDNSSidecarSpec')]",
     "kubernetesHeapsterSpec": "[parameters('kubernetesHeapsterSpec')]",
     "kubernetesMetricsServerSpec": "[parameters('kubernetesMetricsServerSpec')]",
+{{if IsNVIDIADevicePluginEnabled}}
```

Comment:
is this one different?

Reply:
I actually don't know the order of precedence here. Both the method on `Properties` and the function inside the template funcMap have the same name. I'm kind of hesitant to poke around at present.

Reply:
Oh I see that function was already there... It's the same thing as the others we don't need it but since it's already implemented it's already used in other places so makes sense to fix it separately (a bunch of other template funcMap functions are probably unnecessary as well so we could remove all of them in one PR later)